app/controllers/api/v1/scholarship_applications_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  module V1
+    class ScholarshipApplicationsController < ApplicationController
+      respond_to :json
+
+      def create
+        @scholarship_application = ScholarshipApplication.new(reason: params[:reason], terms_accpeted: params[:terms_accepted], scholarship_id: params[:scholarship_id], user_id: current_user.id)
+        if @scholarship_application.save
+          render json: { redirect_to: '/success' }
+        else
```

Comment:
Instead of the conditional (`else`) here, you can use the `create!` with a `!`, and then `rescue` any errors, and `render` them to the frontend.

Here is an example of that in the [`api/v1/team_members_controller`](https://github.com/OperationCode/operationcode_backend/blob/master/app/controllers/api/v1/team_members_controller.rb#L10-L16).